docs/configuration.md

```diff
@@ -1730,6 +1730,126 @@ will apply. It will also use different parameters for test suites.
 
 > This parameter does not modify which files are formatted.
 
+## Spaces
+
+### `spaces.beforeContextBoundColon`
+
+```scala mdoc:defaults
+spaces.beforeContextBoundColon
+```
+
+```scala mdoc:scalafmt
+spaces.beforeContextBoundColon=Never
+---
+def method[A: Bound]: B
+def method[A : Bound]: B
+def method[A: Bound: Bound2]: B
+```
+
+```scala mdoc:scalafmt
+spaces.beforeContextBoundColon=Always
+---
+def method[A: Bound]: B
+def method[A : Bound]: B
+def method[A: Bound: Bound2]: B
+```
+
+```scala mdoc:scalafmt
+spaces.beforeContextBoundColon=IfMultipleBounds
+---
+def method[A: Bound]: B
+def method[A : Bound]: B
+def method[A: Bound: Bound2]: B
+```
+
+### `spaces.afterTripleEquals`
```

Comment:
I have doubts about the rationality of support special formatting for method not even from standard library.

Reply:
@kitbellew @olafurpg what do you think, is it really necessary?

Reply:
> i don't really know how it came about. some colleagues from the spark world know, perhaps?

scalatest and [cats](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/syntax/eq.scala#L12) also uses it.

> if we want to deprecate, then i believe the usual algorithm is to mark deprecated and wait 2 years. :)

Yes, it will be good

Reply:
Removed it from the docs until we decided what would happen to it next